Bottom: b63e1b7f97abb8d778caec62a5c10a24b6bc80a5
Top:    dcff86e5e86aeafe8735074d5254525b5b68703e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-18 16:33:43 -0500

Refresh of svn-haproxy

---
diff --git a/docker/docker-svn/config/000-default.conf b/docker/docker-svn/config/000-default.conf
index 749e5e6da..0fd74ecd7 100644
--- a/docker/docker-svn/config/000-default.conf
+++ b/docker/docker-svn/config/000-default.conf
@@ -1,5 +1,8 @@
 <VirtualHost *:80>
-    ServerAdmin svn_webmaster@SVN_HOSTNAME
-    ServerName SVN_HOSTNAME
-    Redirect / https://SVN_HOSTNAME/
+    ServerAdmin svn_webmaster@svn.novatech-llc.com
+    ServerName svn.novatech-llc.com
+    DocumentRoot /var/www/html/
+
+    ErrorLog /proc/self/fd/2
+    CustomLog /proc/self/fd/1 combined
 </VirtualHost>
diff --git a/docker/docker-svn/config/000-websvn.conf b/docker/docker-svn/config/000-websvn.conf
index 51cefd508..269a4b1de 100644
--- a/docker/docker-svn/config/000-websvn.conf
+++ b/docker/docker-svn/config/000-websvn.conf
@@ -20,13 +20,13 @@ Alias /temp     /var/cache/websvn/tmp
 Alias /wsvn.php /etc/websvn/wsvn.php
 
 <Directory /usr/share/websvn>
-    Options FollowSymLinks 
+    Options FollowSymLinks
     AllowOverride None
     AuthType Basic
     AuthBasicProvider ldap
     AuthName "svn repository"
     AuthLDAPURL "ldap://ldap/ou=user,dc=novatech?uid?sub?(objectClass=Person)"
-    AuthLDAPBindAuthoritative off
+    AuthLDAPBindAuthoritative on
     AuthLDAPSearchAsUser on
     AuthLDAPCompareAsUser on
     AuthLDAPBindDN cn=proxyagent,dc=novatech
@@ -35,8 +35,8 @@ Alias /wsvn.php /etc/websvn/wsvn.php
     AuthLDAPGroupAttributeIsDN off
     <RequireAll>
         Require valid-user
-        Require ssl
-        Require ip 172.16.0.0/16 192.168.0.0/16
+        # Require ssl
+        # Require ip 172.16.0.0/16 192.168.0.0/16
         <RequireAny>
             Require ldap-group cn=%{SERVER_NAME},ou=group,dc=novatech
             Require ldap-group cn=_websvn.UI,cn=%{SERVER_NAME},ou=group,dc=novatech
diff --git a/docker/docker-svn/config/ldap.conf b/docker/docker-svn/config/ldap.conf
index 7fa7ffc76..6ba52f119 100644
--- a/docker/docker-svn/config/ldap.conf
+++ b/docker/docker-svn/config/ldap.conf
@@ -20,8 +20,8 @@ LDAPOpCacheTTL 600
     AuthLDAPGroupAttributeIsDN off
     <RequireAll>
         Require valid-user
-        Require ssl
-        Require ip 172.16.0.0/16 192.168.0.0/16
+        # Require ssl
+        # Require ip 172.16.0.0/16 192.168.0.0/16
         Require ldap-group cn=%{SERVER_NAME},ou=group,dc=novatech
         Require ldap-group cn=ldap.novatech-llc.com,ou=group,dc=novatech
     </RequireAll>
diff --git a/docker/docker-svn/docker-entrypoint.sh b/docker/docker-svn/docker-entrypoint.sh
index 1da39e43c..7e5d50436 100755
--- a/docker/docker-svn/docker-entrypoint.sh
+++ b/docker/docker-svn/docker-entrypoint.sh
@@ -66,8 +66,8 @@ case ${1} in
     AuthLDAPGroupAttributeIsDN off
     <RequireAll>
         Require valid-user
-        Require ssl
-        Require ip 172.16.0.0/16 192.168.0.0/16
+        # Require ssl
+        # Require ip 172.16.0.0/16 192.168.0.0/16
         <RequireAny>
             Require ldap-group cn=%{SERVER_NAME},ou=group,dc=novatech
             Require ldap-group cn=${repo_name},cn=%{SERVER_NAME},ou=group,dc=novatech
